common/src/main/java/com/github/twitch4j/common/util/EscapeUtils.java

```diff
@@ -70,14 +70,14 @@ public static String escapeTagValue(Object value) {
     public static String unescapeTagValue(String value) {
         return StringUtils.replaceEach(
             value,
-            new String[] {
+            new String[]{
```

Comment:
I personally [prefer](https://google.github.io/styleguide/javaguide.html#s4.8.3.1-array-initializers) the space but we can remove it